CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ Main (unreleased)
 
 - riverfmt: Allow function arguments to persist across different lines. (@rfratto)
 
+- New Grafana Agent Flow components:
+
+  - `remote.http` polls an HTTP URL and exposes the response as a string or
```

Comment:
Should this be exposes the `body` instead of the response? My mind goes to response codes.

Reply:
yep, good point